buildcraft_resources/assets/buildcraftfactory/compat/buildcraft/guide/en_us/block/pump.xml

```diff
@@ -0,0 +1,15 @@
+<chapter name="tile.pumpBlock.name"/>
+<lore>
+With the collection of fluids by hand can often be quite tedious and dangerous at times, the need for an automated way to gather fluids was required.
+</lore>
+<no_lore>
+The pump is a way to automatically pick up any fluid sources found directly below it and any sources connected to it.
+</no_lore>
+<chapter name="Information"/>
+To gather fluids from the world, place a pump ontop of the highest fluid source block of the source you want to pump up and supply it with power.
```

Comment:
doesn't have to be the highest, if you put it on the lowest it'll suck up the higher ones as well (as if it pumps up below it and the higher fluid flows downwards to fill the void)